devsetup/Makefile

```diff
@@ -5,7 +5,7 @@ CRC_DEFAULT_NETWORK_IP ?= 192.168.122.10
 EDPM_COMPUTE_SUFFIX ?= 0
 EDPM_TOTAL_NODES ?= 1
 EDPM_COMPUTE_IP ?= 192.168.122.100
-OPENSTACK_RUNNER_IMG ?= quay.io/openstack-k8s-operators/openstack-ansibleee-runner:latest
+DATAPLANE_RUNNER_IMG ?= quay.io/openstack-k8s-operators/openstack-ansibleee-runner:latest
```

Comment:
`OPENSTACK_RUNNER_IMG` is currently being used by `edpm_play` target but I don't know if it should still exist TBH.

Reply:
I'll change it.